docs/csharp/tour-of-csharp/tutorials/list-collection.md

```diff
@@ -1,25 +1,55 @@
 ---
-title: Data collections - Introductory interactive tutorial
-description: In this tutorial, you use your browser to learn about C# collections. You write C# code and see the results of compiling and running your code directly in the browser.
-ms.date: 03/07/2025
+title: Data collections - Introductory tutorial
+description: In this tutorial, you use GitHub Codespaces to learn about C# collections. You write C# code and see the results of compiling and running your code in the Codespaces environment.
+ms.date: 12/10/2025
 ---
-# Learn to manage data collections using List\<T> in C\#
+# Tutorial: Learn to manage data collections using List\<T> in C\#
 
-This introductory tutorial provides an introduction to the C# language and the basics of the  class.
+This introductory tutorial provides an introduction to the C# language and the basics of the <xref:System.Collections.Generic.List`1?displayProperty=nameWithType> class.
 
-This tutorial teaches you C# interactively, using your browser to write C# code and see the results of compiling and running your code. It contains a series of lessons that create, modify, and explore collections and arrays. You work primarily with the <xref:System.Collections.Generic.List%601> class.
+This tutorial teaches you C#. You write C# code and see the results of compiling and running that code. It contains a series of lessons that create, modify, and explore collections and arrays. You work primarily with the <xref:System.Collections.Generic.List%601> class.
```

Comment:
```suggestion
This tutorial teaches you C#. You write C# code and see the results of compiling and running that code. It contains a series of lessons that create, modify, and explore collections. You work primarily with the <xref:System.Collections.Generic.List%601> class.
```

Reply:
This is the only time in the article that arrays are mentioned. Seems out of place.